tests/provenance/java-pointsto/java-pointsto.dl

```diff
@@ -0,0 +1,177 @@
+// Souffle - A Datalog Compiler
+// Copyright (c) 2013, Oracle and/or its affiliates. All rights reserved
+// Licensed under the Universal Permissive License v 1.0 as shown at:
+// - https://opensource.org/licenses/UPL
+// - <souffle root>/licenses/SOUFFLE-UPL.txt
+
```

Comment:
Include an explanation of what this test is for.

Also consider replacing the test with a minimal input that tests just the desired behaviour.